### PR TITLE
Fix completion window and topic name

### PR DESCRIPTION
**Pull request by yatharthranjan**

### CODE REVIEW

File: RALPMH-ILD-Lon-s1/protocol.json [hunk 1]
```diff
@@ -295,7 +295,7 @@
       "order": 3,
       "questionnaire": {
         "repository": "https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/",
-        "name": "pulse_ox_response",
+        "name": "pulse_ox",
```
Comment (mpgxvii) [inline RALPMH-ILD-Lon-s1/protocol.json:298]: I think this should be the name of the questionnaire in the definitions.

**Review by mpgxvii — APPROVED** [review]
LGTM

File: RALPMH-ILD-Lon-s1/protocol.json (hunk 1 above)
Comment (yatharthranjan) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: yes it is -- https://github.com/RADAR-base/RADAR-Schemas/blob/4fc1e97ee7322e070b08549d4e20d291cb8630de/specifications/active/aRMT-1.14.0.yml#L201

**Review by afolarin — APPROVED** [review]

File: RALPMH-ILD-Lon-s1/protocol.json (hunk 1 above)
Comment (mpgxvii) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: Sorry I meant the name in the url: `master/questionnaires/pulse_ox_response/pulse_ox_response_armt.json` Cause this name is what's used to find the Github url. Which should also match the `type` name in Schemas.

Comment (yatharthranjan) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: So it will not use the `questionnaire_definition_url` from the specs to get the definition?

Comment (mpgxvii) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: No it uses the one here in the protocol.

Comment (yatharthranjan) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: hmm, ok then I can rename the questionnaire in redcap and run the script.

Comment (mpgxvii) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: Okay sure thanks

Comment (yatharthranjan) [reply RALPMH-ILD-Lon-s1/protocol.json:298]: It is available on master now, proceeding with the merge

**Pull request merged**